src/sqlfluff/cli/commands.py

```diff
@@ -274,8 +274,13 @@ def dialects(**kwargs):
         "found. This is potentially useful during rollout."
     ),
 )
+@click.option(
+    "--not-ignore-files",
```

Comment:
Again I don't quite understand this from first glance. What do you think of something more explicit like

```suggestion
    "--disregard-sqlfluffignores",
```